repose-aggregator/services/phone-home/src/test/scala/org/openrepose/core/services/phonehome/PhoneHomeServiceTest.scala

```diff
@@ -426,12 +430,17 @@ class PhoneHomeServiceTest extends FunSpec with Matchers with MockitoSugar {
 
       phoneHomeService.SystemModelConfigurationListener.configurationUpdated(systemModel)
 
+      val stringArgumentCaptor = ArgumentCaptor.forClass(classOf[String])
       verify(mockAkkaServiceClient).post(
         anyString(),
         mockitoEq(collectionUri),
         anyMapOf(classOf[String], classOf[String]),
-        mockitoEq(expectedMessage),
+        stringArgumentCaptor.capture,
         mockitoEq(MediaType.APPLICATION_JSON_TYPE))
+      val postBody = stringArgumentCaptor.getValue.toString
+      assert("""\{"createdAt":"[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}[+-][0-9]{2}:[0-9]{2}","createdAtMillis":[0-9]{13},"jreVersion":".*","jvmName":".*","serviceId"""".r
+        .findFirstIn(postBody).isDefined)
+      assert(postBody.endsWith(expectedMessage.substring(1)))
```

Comment:
I did this, but I'm not sure it made it any cleaner. On the other hand, it didn't make it any worse. ;-)